packages/coordinator/src/services/translation/request/withdraw/withdraw.ts

```diff
@@ -21,19 +27,34 @@ export function convertTaskToEtpWithdraw(task: fhir.Task): hl7V3.EtpWithdraw {
   const practitionerRole = getContainedPractitionerRoleViaReference(task, practitionerRoleRef.reference)
 
   const organizationRef = practitionerRole.organization
-  if(!isReference(organizationRef)) {
+  if (!isReference(organizationRef)) {
     throw new processingErrors.InvalidValueError(
       "practitionerRole.organization should be a Reference",
       'task.contained("PractitionerRole").organization'
     )
   }
 
+  const repeatInformation = getExtensionForUrl(
+    task.extension,
+    "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation",
+    'Task.extension("EPS-Repeat-Information")'
+  ) as fhir.ExtensionExtension<fhir.IntegerExtension>
+
+  const numberOfRepeatsIssuedExtension = getExtensionForUrl(
+    repeatInformation.extension,
+    "numberOfRepeatsIssued",
+    `Task.extension("https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation").extension`
+  ) as fhir.IntegerExtension
+
+  const repeatNumber = numberOfRepeatsIssuedExtension.valueInteger.toString()
```

Comment:
Would we expect pertinentInfo1 to still be added with a default value of 1? Or just not exist at all, just pushed up a change with it not existing at all but now realising it could just use a default of 1 repeats issued?

Reply:
MIM says pertinentInformation1 is 0..1 so that says to me don't include it if it's not relevant.

Reply:
Nice one then these changes should suffice that.